balancer-js/examples/pools/composable-stable/join.ts

```diff
@@ -0,0 +1,99 @@
+import dotenv from "dotenv";
+import { Network } from "@/lib/constants";
+import { BalancerSDK } from "@/modules/sdk.module";
+import { ethers } from "hardhat";
+import pools_14717479 from "@/test/lib/pools_14717479.json";
```

Comment:
Can we separate this out from the other imports and just tidy/reorder the imports in general.